packages/browser/test/e2e/test/onunhandledrejection.test.js

```diff
@@ -0,0 +1,241 @@
+const { test, expect } = require('@playwright/test');
+
+const { getSentryEvents } = require('./utils/helpers');
+
+test.describe('window.onunhandledrejection', () => {
+  test.beforeEach(async ({ baseURL, page }) => {
```

Comment:
this is being repeated on every test file so far. If this is a requirement to run tests properly is there a global way to apply this?

Reply:
I'm moving this to the fixtures 👍